test/tests/functional/pbs_job_array_comment.py

```diff
@@ -84,17 +84,18 @@ def test_job_array_comment(self):
             ATTR_comment: (MATCH_RE, 'Job Array Began at .*')
         }
         self.server.expect(JOB, attr, id=jid, attrop=PTL_AND)
-        self.server.expect(JOB, {ATTR_comment: 'Subjob failed'},
-                           id=create_subjob_id(jid, 0), extend='x')
-        self.server.expect(JOB, {ATTR_comment: 'Subjob failed'},
-                           id=create_subjob_id(jid, 5), extend='x')
-        self.server.expect(JOB, {ATTR_comment: 'Subjob failed'},
-                           id=create_subjob_id(jid, 7), extend='x')
-
-
-def create_subjob_id(job_array_id, subjob_index):
-    """
-    insert subjob index into the square brackets of job array id
-    """
-    idx = job_array_id.find('[]')
-    return job_array_id[:idx+1] + str(subjob_index) + job_array_id[idx+1:]
+        self.server.expect(JOB, {ATTR_comment: 'Not Running: PBS Error:' +
+                                 ' Execution server rejected request' +
+                                 ' and failed'},
+                           id=test_job_array.create_subjob_id(jid, 0),
+                           extend='x')
+        self.server.expect(JOB, {ATTR_comment: 'Not Running: PBS Error:' +
+                                 ' Execution server rejected request' +
+                                 ' and failed'},
+                           id=test_job_array.create_subjob_id(jid, 5),
+                           extend='x')
+        self.server.expect(JOB, {ATTR_comment: 'Not Running: PBS Error:' +
+                                 ' Execution server rejected request' +
+                                 ' and failed'},
+                           id=test_job_array.create_subjob_id(jid, 7),
+                           extend='x')
```

Comment:
Make sure that these messages are part of EDD.

Reply:
EDD was updated 2 days ago and announcement was made in forum
http://community.pbspro.org/t/pp-479-running-subjobs-to-be-able-to-survive-a-pbs-server-restart/835/38?u=shrini-h